src/job_cache/job-cache.wake

```diff
@@ -18,9 +18,7 @@ package build_wake
 from wake import _
 from gcc_wake import _
 
-target jobCacheLib variant = match variant
-    Pair "wasm-cpp14-release" _ -> Pass (SysLib "" Nil Nil Nil Nil)
```

Comment:
Can you cleanup `fuse-waked.wake` too? Maybe just generally revert this PR. https://github.com/sifive/wake/pull/983/files

Will make it easier later if we don't leave cruft around

Reply:
sorry just not found this, I'll do this in a follow up PR, I promise.